plugin.audio.itvrs_radio_ice/default.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python
+
+import os
+import xbmcaddon
+import xbmcgui
+
+addon = xbmcaddon.Addon()
+addonname = addon.getAddonInfo('name')
+line1 = 'ITVRS - Radio Ice'
+line2 = '  '
+line3 = 'Use Left Menu To Pause Or Stop'
+
+xbmcgui.Dialog().ok(addonname, line1, line2, line3)
+
+class itvrs_radio_ice:
+
+    def __init__(self):
+		
+        xbmc.Player().play('http://stream.zeno.fm/3611cwn8mbruv')

```

Comment:
Calling `xbmc.Player().play(...)` isn't allowed in an plugin.
You have to create a directory listing, which contains one `ListItem`. For that list item you set the path to the above URL.